src/core/lib/iomgr/ev_poll_posix.cc

```diff
@@ -60,6 +60,19 @@ typedef struct grpc_fd_watcher {
   grpc_fd* fd;
 } grpc_fd_watcher;
 
+typedef struct grpc_cached_wakeup_fd grpc_cached_wakeup_fd;
+
+/* Only used by poll when GRPC_ENABLE_FORK_SUPPORT=1 */
```

Comment:
The comment "only used by poll" is confusing. Can we remove "by poll" part?

Reply:
Done